lya/analysis.py

```diff
@@ -18,9 +30,18 @@ def residuals(pars, data):
     pars : array-like
         [alpha, beta, gamma] parameters.
 
-    data : Data object
+    data : Data object (string)
         Data to be compared with model.
     """
+    
+    alpha, beta, gamme = pars
+    
+    
+    mod = model(data.k,alpha,beta,gamma)
+    res = data.P_k - mod
```

Comment:
following my previous comment, go ahead and keep these referring to `data.x` and `data.y` rather than `k`/`P_k`